.kokoro/verify-docker-images.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Script to verify the presence of GraalVM docker test images tagged with the latest java shared config version.
+
+javaSharedConfigVersion=1.5.7 # {x-version-update:java-shared-config:current}
```

Comment:
Alternative that doesn't rely on a tool to keep this version in sync:
```
javaSharedConfigVersion=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
```

Reply:
Thanks @burkedavison! Since we are trying to block the release on the presence of the docker image, this value needs to be in sync with the version we are about to release. Correct me if I'm wrong but I believe this command gets us the current version?

Reply:
It gets us the version in source for the current commit. I may be not understanding the requirement here, so feel free to pass on this suggestion.

Reply:
I see! Thank you, you're right, just verified with running this command in https://github.com/googleapis/java-shared-config/pull/678. Applied this change. It is a much neater solution than relying on a bot.